From c5e1ab57082b7fa8b9fa6dd6ab9911819d63176c Mon Sep 17 00:00:00 2001
From: "guoyin.chen" <guoyin.chen@freescale.com>
Date: Mon, 10 Nov 2014 10:02:21 +0800
Subject: [PATCH 379/602] MA-5839-1 Update android to Lollipop

Add below two envs to check the boot image&recovery image size
Set the max size to 8MB to align the MFG tool setting
BOARD_BOOTIMAGE_PARTITION_SIZE := 8388608
BOARD_RECOVERYIMAGE_PARTITION_SIZE := 8388608

Signed-off-by: guoyin.chen <guoyin.chen@freescale.com>
---
 imx6/BoardConfigCommon.mk | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/imx6/BoardConfigCommon.mk b/imx6/BoardConfigCommon.mk
index 5d558db..3d73b86 100644
--- a/imx6/BoardConfigCommon.mk
+++ b/imx6/BoardConfigCommon.mk
@@ -57,7 +57,8 @@ ARCH_ARM_USE_NON_NEON_MEMCPY := true
 # comment out for 1g/3g space split
 # TARGET_KERNEL_2G := true
 
-BOARD_RECOVERY_PARTITION_SIZE = 10M
+BOARD_BOOTIMAGE_PARTITION_SIZE := 8388608
+BOARD_RECOVERYIMAGE_PARTITION_SIZE := 8388608
 
 BOARD_SYSTEMIMAGE_PARTITION_SIZE := 377487360
 BOARD_FLASH_BLOCK_SIZE := 4096
-- 
1.8.0

